homeassistant/components/tuya/__init__.py

```diff
@@ -137,6 +138,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     ) as exc:
         raise ConfigEntryNotReady() from exc
 
+    except TuyaAPIRateLimitException as exc:
+        _LOGGER.error("Tuya login ratelimited")
+        raise ConfigEntryNotReady() from exc
+
```

Comment:
I think that this can be grouped with others exceptions and you don't need the log, you already have `from exc`

Reply:
I personal like the log, but am open to removing it. The current error codes (and translations) don't really tell the user that this is a transient error. The logs make this more clear

Reply:
oh, I miss-read what you mean. The `TuyaAPIRateLimitException` is stacked with `ConfigEntryNotReady`. Let me see if that gets unwrapped in logs.

Reply:
hrm, config_entries does not print out the actual exception, or stack. I'd vote for keeping the log, but if you want to remove it I'll change it.

Reply:
If you want to leave it in my opinion should be a warning, but let @MartinHjelmare suggest best solution based on best practice.

Reply:
It's good to keep the log.